build-logic-commons/src/main/kotlin/gradlebuild.code-quality.gradle.kts

```diff
@@ -77,7 +77,7 @@ checkstyle {
     toolVersion = "8.12"
     config = configFile("checkstyle.xml")
     val projectDirectory = layout.projectDirectory
-    configDirectory.set(rules.elements.map { projectDirectory.dir(it.single().asFile.absolutePath).dir("checkstyle") })
+    configDirectory.set(rules.elements.map { projectDirectory.dir(it.first().asFile.absolutePath).dir("checkstyle") })
```

Comment:
Previously `code-quality-rules` only contains 1 entry, but now it's `build-logic-commons` which contains a list of entries.

Reply:
We talk about this and said that for now the `code-quality-rules` should stay a separate subproject.